packages/protocol/contracts/governance/DowntimeSlasher.sol

```diff
@@ -5,22 +5,32 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 import "./SlasherUtil.sol";
 
+/**
```

Comment:
nit: Please use the natspec format defined here:
https://solidity.readthedocs.io/en/v0.6.11/natspec-format.html